meshapp/src/home/logged-out/home.tsx

```diff
@@ -59,7 +67,7 @@ export default function LoggedOutPage() {
               filter: "drop-shadow(4px 4px 3px rgba(0,0,0,.6))",
             }}
           >
-            {LoginInput()}
+            {showForgotPassword ? <ForgotPassword updateShowForgotPasswordState={updateShowForgotPasswordState} /> : <LoginInput updateShowForgotPasswordState={updateShowForgotPasswordState} />}
```

Comment:
This seems repetitive.  Is there anyway to incorporate it into login?

Reply:
Refactored :)